tell_me_your_secrets/__main__.py

```diff
@@ -28,14 +31,15 @@
 tell-me-your-secrets <PATH_TO_FOLDER> -f aws microsoft crypto digitalocean ssh sql google
 
 '''.format(config_names=config_names)
-arguement_parser = argparse.ArgumentParser(description=module_description)
-arguement_parser.formatter_class = argparse.RawDescriptionHelpFormatter
-arguement_parser.add_argument('search_path',help='The Root Directory From which the Search for the Key/Pem files is initiated')
-arguement_parser.add_argument('-c','--config',help='Path To Another config.yml for Extracting The Data')
-arguement_parser.add_argument('-w','--write',help='Path of the csv File to which results are written')
-arguement_parser.add_argument('-f','--filter',help='Filter the Signatures you want to apply. ',nargs='+')
-arguement_parser.add_argument('-v','--verbose',help='Enable debug level logging. ', action='store_true')
-arguement_parser.add_argument('-e','--exit',help='Exit non-zero on results found. ',action='store_true')
+argument_parser = argparse.ArgumentParser(description=module_description)
+argument_parser.formatter_class = argparse.RawDescriptionHelpFormatter
+argument_parser.add_argument('search_path', help='The Root Directory From which the Search for the Key/Pem files is initiated')
+argument_parser.add_argument('-c', '--config', help='Path To Another config.yml for Extracting The Data')
+argument_parser.add_argument('-w', '--write', help='Path of the csv File to which results are written')
+argument_parser.add_argument('-f', '--filter', help='Filter the Signatures you want to apply. ', nargs='+')
+argument_parser.add_argument('-v', '--verbose', help='Enable debug level logging. ', action='store_true')
+argument_parser.add_argument('-e', '--exit', help='Exit non-zero on results found. ', action='store_true')
+argument_parser.add_argument('-g', '--gitignore', help='Ignore .gitignore mapped objects. ', action='store_true')
```

Comment:
@valayDave I'm really not sure on the short name for this one

Thinking about the previous one I did for exit on non-zero (`-e`) I feel that could have been best suited for when an entropy check is added.

Reply:
Can't we just make it `--en` when we add entropy checks?

Reply:
True - What do you think about `-g` for gitignore or would `-gi` make more sense?